### PR TITLE
github-practice-for-all

### PR DESCRIPTION
**Pull request by ghost**
For new coders to experience team workflow and practice github

### CODE REVIEW

File: _data/projects/github-practice-for-all.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: Github-practice-for-all
+desc: Here you are welcome to test how team workflow might look and practice git pull, git push and more...
+site: https://github.com/RomchyFCC/GitHub-Practice-For-All
+tags:
+- github
+- practice
+- pull
+- push
+upforgrabs:
+  name: add-name-to-file
```
Comment (ritwik12) [inline _data/projects/github-practice-for-all.yml]: @RomchyFCC Please change it to up-for-grabs

Comment (ghost) [reply _data/projects/github-practice-for-all.yml]: done, now it failed the test, what's up?

File: _data/projects/github-practice-for-all.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: Github-practice-for-all
+desc: Here you are welcome to test how team workflow might look and practice git pull, git push and more...
+site: https://github.com/RomchyFCC/GitHub-Practice-For-All
+tags:
+- github
+- practice
+- pull
+- push
+up-for-grabs:
```
Comment (ritwik12) [inline _data/projects/github-practice-for-all.yml]: @RomchyFCC It is "upforgrabs:".

File: _data/projects/github-practice-for-all.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+name: Github-practice-for-all
+desc: Here you are welcome to test how team workflow might look and practice git pull, git push and more...
+site: https://github.com/RomchyFCC/GitHub-Practice-For-All
+tags:
+- github
+- practice
+- pull
+- push
+up-for-grabs:
+  name: add-name-to-file
```
Comment (ritwik12) [inline _data/projects/github-practice-for-all.yml]: @RomchyFCC Please change it to "up-for-grabs" as your label name

Comment (ghost) [reply _data/projects/github-practice-for-all.yml]: Done, ty

File: _data/projects/github-practice-for-all.yml (hunk 1 above)
Comment (ritwik12) [reply _data/projects/github-practice-for-all.yml]: @RomchyFCC Your label name is still "add-name-to-file" whereas it should be "up-for-grabs" as in link.

**Comment by ritwik12** [comment]
@RomchyFCC Thanks!!

**Comment by ghost** [comment]
@ritwik12 Thank YOU! have a nice weekend!

**Comment by ritwik12** [comment]
@RomchyFCC Thanks, You too :)

**Pull request merged**